sdk/appconfiguration/app-configuration/recordings/node/authentication/recording_invalid_connection_string_gives_a_decent_error_message.js

```diff
@@ -1,5 +1,5 @@
 let nock = require('nock');
 
-module.exports.hash = "1f8d0464882356949ec20fb9901bed1a";
+module.exports.hash = "b8a3dd917708abcc6af0b0637d9297ac";
 
-module.exports.testInfo = {"uniqueName":{"":"158700300978708132"},"newDate":{}}
+module.exports.testInfo = {"uniqueName":{},"newDate":{}}
```

Comment:
Do we need to record this again?

Reply:
Test might have changed, we don't need it anymore.